cypress/integration/deskbook-dropdown.js

```diff
@@ -0,0 +1,36 @@
+describe("Update deskbook version to 2018 ", function() {
+
+
+  beforeEach(() => {
+    const email = Cypress.env("email");
+    const password = Cypress.env("password");
+    cy.visit("localhost:8100");
+
+    // it is ok for the email to be visible in the Command Log
+    expect(email, "email was set").to.be.a("string").and.not.be.empty;
+    // but the password value should not be shown
+    if (typeof password !== "string" || !password) {
+      throw new Error("Missing password value, set using CYPRESS_password=...");
+    }
+
+    cy.get('input[name="emaial"]')
```

Comment:
Refactor this with the login custom command here

Reply:
Completed